### PR TITLE
ADAPT-4307: Added unsafe-eval to Netlify security policy to allow AdPlugg embeds.

### PR DESCRIPTION
**Pull request by nplowman**
# READY FOR REVIEW

# Summary
- Adds 'unsafe-eval' directive to content security policy, to allow for embedding of AdPlugg ads.

# Review By (Date)
- Friday, Feb. 11th.

# Criticality
- 7/10? Issue does not affect anything currently on production, but is blocking rollout of a new feature.

# Review Tasks

## Setup tasks and/or behavior to test

1. Visit the following test page, where an embedded ad is present: (link removed)
2. Validate that the ad is displaying, and that the content-security-policy error is no longer in the console.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   for = "/*"
   [headers.values]
     Access-Control-Allow-Origin = "*"
-    Content-Security-Policy = "default-src https: 'unsafe-inline'; form-action https:; img-src https: data:"
+    Content-Security-Policy = "default-src https: 'unsafe-inline' 'unsafe-eval'; form-action https:; img-src https: data:"
```
Comment (sherakama) [inline netlify.toml:18]: We better be careful who we embed 😟 . This does seem necessary as we do allow are content authors to embed from anywhere.

**Review by sherakama — APPROVED** [review]
~GTG~

@nplowman the example URL is still not loading for me. Postscript is complaining about something... 

Screen Shot 2022-02-07 at 4 19 30 PM

**Review by sherakama — CHANGES_REQUESTED** [review]
Script errors are happening

**Comment by sherakama** [comment]
@nplowman please review the feedback

**Comment by nplowman** [comment]
@sherakama Hmm. That's odd - I'm not able to reproduce this on my end. Do you happen to have an ad blocker enabled, or any other chrome plugins that could be interfering? 

Could you see if you get the same issue in an incognito window?

**Comment by sherakama** [comment]
Ah. That seems to be it. Sorry for the false alarm.

**Review by sherakama — APPROVED** [review]

**Pull request merged**